packages/build/tests/core/tests.js

```diff
@@ -320,3 +322,30 @@ if (!version.startsWith('v8.')) {
     await runFixture(t, 'functions_user_missing')
   })
 }
+
+test('Does not generate a `manifest.json` file when the feature flag is not enabled', async (t) => {
+  const fixtureName = 'functions_internal_no_manifest_2'
+
+  await removeDir(`${FIXTURES_DIR}/${fixtureName}/.netlify/functions`)
+  await runFixture(t, fixtureName, { flags: { mode: 'buildbot' } })
```

Comment:
Should `snapshot: false` be used in those 3 tests, since we are mostly interested by checking the generated `manifest.json`?

Reply:
In https://github.com/netlify/build/pull/3378/commits/6ec24c720770975f6deb3a2864bc95ca0964e213, I've added `snapshot: false` to the 2 tests that don't create a manifest file, but left it for the test that does, just to assert that the creation of the manifest file does not affect the output. What do you think?

Reply:
Sounds good!